### ISSUE TITLE
em dash seems to be replaced with minus

### ISSUE THREAD

**Issue by Jun711** [spencermountain/compromise#638]
with compromise v12.0.0, `em dash` seems to be replaced with `minus`. 

This is how it was used.
```
let nlpObj = nlp('text—text');
const sen = r.sentences().data();
```

**Comment by spencermountain** [comment]
ah! (very!) good eye.
thank you. I'll take a look at this this week.

**Comment by spencermountain** [comment]
hey @Jun711 this is fixed now in `12.2.0`
cheers
